SplitPDF/src/main/kotlin/com/datalogics/pdfl/samples/SplitPDF.kt

```diff
@@ -0,0 +1,40 @@
+package com.datalogics.pdfl.samples
+
+import com.datalogics.PDFL.*
+import java.util.*
+
+/*
+* This program opens a PDF input document and exports the pages to a set of separate PDF documents.
+*
+* Copyright (c) 2024, Datalogics, Inc. All rights reserved.
+*/
+
+fun main(args: Array<String>) {
+    println("SplitPDF sample:")
+    val lib = Library()
+
+    try {
+        val sInput: String = Library.getResourceDirectory() + "Sample_Input/PDFToBeSplit.pdf"
```

Comment:
Where is "PDFToBeSplit.pdf"?

Reply:
The PDF document should be at `target\lib\Resources\Sample_Input\`.